recipe/patches/0005-osx-arm64.patch

```diff
@@ -1,5 +1,25 @@
+From d1b4f86fc52acf2701bb2b3843352b0e3bb2d7e2 Mon Sep 17 00:00:00 2001
+From: "Uwe L. Korn" <uwe.korn@quantco.com>
+Date: Thu, 1 Apr 2021 12:43:34 +0200
+Subject: [PATCH 5/5] osx-arm64
```

Comment:
Combined both osx-arm64 patches, because there didn't seem to be a strong reason for them to be separate; message taken from the file-name of the patch